frontend/src/layout/FeaturePreviews/featurePreviewsLogic.tsx

```diff
@@ -26,7 +26,11 @@ export const featurePreviewsLogic = kea<featurePreviewsLogicType>([
         actions: [supportLogic, ['submitZendeskTicket']],
     })),
     actions({
-        updateEarlyAccessFeatureEnrollment: (flagKey: string, enabled: boolean) => ({ flagKey, enabled }),
+        updateEarlyAccessFeatureEnrollment: (flagKey: string, enabled: boolean, stage?: string) => ({
+            flagKey,
+            enabled,
+            stage,
+        }),
```

Comment:
minor, but should `stage` be an enum instead of a string?

Reply:
Looks like there's an existing enum:

```ts
export enum EarlyAccessFeatureStage {
    Draft = 'draft',
    Concept = 'concept',
    Alpha = 'alpha',
    Beta = 'beta',
    GeneralAvailability = 'general-availability',
    Archived = 'archived',
}
```

I noticed that in `posthog-js` there's also an enum, but it has less options:

```ts
export type EarlyAccessFeatureStage = 'concept' | 'alpha' | 'beta' | 'general-availability'
```

Seems to me that these two should be the same and we should make use the enum type on both sides. Or should we let the js version continue to accept a string and only make the type change here?